### ISSUE TITLE
Caption, size, downloadUrl, etc not shown on first uploaded file or when overwriteInitial = true

### ISSUE THREAD

**Issue by nikomiranda** [kartik-v/bootstrap-fileinput#1544]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest `master` branch of bootstrap-fileinput.
- [ ] This is not an usage question. I confirm having read the plugin [documentation](http://plugins.krajee.com/file-input) and [demos](http://plugins.krajee.com/file-input/demo).
- [ ] This is not a general programming / coding question. (Those should be directed to the [webtips Q & A forum](http://webtips.krajee.com/questions)).
- [X] I have attempted to find the simplest possible steps to reproduce the issue.
- [ ] I have included a failing test as a pull request (Optional).

## Steps to reproduce the issue

```javascript
	var $el1 = $("#fileinput");
	$el1.fileinput({
        theme: 'fa',
        autoOrientImage: false,
        uploadUrl: "/fileinput/upload",
        uploadAsync: true,
        enableResumableUpload: false,
        minFileCount: 1,
        maxFileCount: 1,
        allowedFileTypes: ['image'],    // allow only images
        initialPreviewAsData: true,
        overwriteInitial: false,
        showCancel: false,
        showClose: false,
        showUpload: false, // hide upload button
        showRemove: false, // hide remove button
        showCaption: false,
        showUploadStats: true,
        browseOnZoneClick: true,
		"fileActionSettings":{
			"showDrag":false,	
		},        
	}).on("filebatchselected", function(event, files) {
    	$el1.fileinput("upload");
    });
```

The Json from server:

```
{  
   "initialPreview":[
      "/uploads/samurai.jpg"
	],
   "initialPreviewConfig":[      
   {
         "caption":"samurai1.jpg",
         "size":11432,
         "downloadUrl":"/uploads/samurai.jpg",
         "key":"1"		
	}
	]
}
```

## Expected behavior and actual behavior

When I follow those steps, I see...

![image](https://user-images.githubusercontent.com/9426688/80479770-7f275e00-898a-11ea-9e65-3646f6d5c659.png)

I was expecting...

## Environment

Browsers

- [ X ] Google Chrome
- [ X ] Mozilla Firefox
- [ ] Internet Explorer
- [ ] Safari

Operating System

- [ X ] Windows
- [ ] Mac OS X
- [ ] Linux
- [ ] Mobile

Libraries

- jQuery version: 3.4.1
- bootstrap-fileinput version: 5.0.8

## Isolating the problem

- [ ] This bug happens [on the plugin demos page](http://plugins.krajee.com/file-input/demo)
- [X] The bug happens consistently across all tested browsers
- [ ] This bug happens when using bootstrap-fileinput without other plugins
- [ ] I can reproduce this bug in [a jsbin](https://jsbin.com/)

**Comment by nikomiranda** [comment]
If overwriteInitial: true then the image always does not shown any caption information .

**Comment by kartik-v** [comment]
Not sure if you have updated. Please check if you have latest version. This issue can not be reproduced on the latest version. If you still have an issue update and will reopen the issue.

**Comment by nikomiranda** [comment]
I confirmed is resolved on 5.1.0
